src/app/pages/home/components/account-info-card.tsx

```diff
@@ -14,7 +15,7 @@ import { AccountActions } from './account-actions';
 
 export function AccountInfoCard() {
   const name = useCurrentAccountDisplayName();
-  const [isNarrowViewport] = useMediaQuery('(max-width: 600px)');
+  const [isNarrowViewport] = useMediaQuery(`(max-width: ${token('sizes.narrowViewPortMaxWidth')})`);
```

Comment:
I was going to refactor this to use `useViewportMaxWidth` and our sized tokens but we have no size for `600px`. 

Maybe `600px` here is outdated and it should be `md`? 

Do we need both `tokens.sizes` & `tokens.breakpoints`?

Reply:
I believe we can use `md` here, or better even delete and just use array:
`mr={['0px' , 'space.02']}`? smth like that

Reply:
I think breakpoints is enough, no?

Reply:
hm, actually there probably might be some edge cases when we need that custom sizes as well, so prob better to keep them both

Reply:
do we have to do the breakpoint at 600? Why not change it to something else?

Reply:
I can test it and see if we can do it at a different size using the `breakpoints` we have, thanks! 

Sometimes we need to use  `tokens.sizes` when specifying `maxWidth` but I think sometimes we are using them incorrectly as breakpoints. 

I'll spend some time checking and cleaning it up, thanks both 👏